rakelib/test.rake

```diff
@@ -30,6 +30,9 @@ namespace :test do
     java_home = [ ENV_JAVA['java.home'], File.join(ENV_JAVA['java.home'], '..') ] # in case of jdk/jre
     javac = java_home.map { |home| File.expand_path('bin/javac', home) }.find { |javac| File.exist?(javac) } || 'javac'
     sh "#{javac} -cp #{classpath} -d test/target/test-classes #{Dir['spec/java_integration/fixtures/**/*.java'].to_a.join(' ')}"
+    # TODO: What about environments without this installed?
+    kotlinc = 'kotlinc'
+    sh "#{kotlinc} -cp #{classpath} -include-runtime -d test/target/test-classes/kotlin_fixtures.jar #{Dir['spec/java_integration/fixtures/**/*.kt'].to_a.join(' ')}"
```

Comment:
I built this as a jar using the `-include-runtime` parameter to ensure that the necessary kotlin libraries would be available when running the tests.  Otherwise, you'll need to arrange to have the kotlin runtime included in the classpath when running the kotlin tests, since the compiled fixtures end up referencing kotlin types.